app/models/manageiq/providers/lenovo/physical_infra_manager/event_catcher/stream.rb

```diff
@@ -54,7 +69,7 @@ def create_event_connection(ems)
                 :port => ems.endpoints.first.port)
   end
 
-  def get_last_cnn_from_events(ems_id)
-    EventStream.where(:ems_id => ems_id).maximum('ems_ref') || 1
+  def last_event_ems_ref(ems_id)
+    EventStream.where(:ems_id => ems_id).maximum('CAST(ems_ref AS int)')
```

Comment:
This is going to be expensive but now that it is only done once on startup it should be manageable.  If this proves to be an issue we can add a column to the events stream table to store this specifically.

Reply:
Thank you. I think all the thinks are ok, but if we need in the future add a new column to store this, let me kwon I can do this.  :+1: